docs/core/runtime-config/garbage-collector.md

```diff
@@ -557,6 +557,7 @@ Example:
 ## Standalone GC
 
 - Specifies the name of a standalone GC DLL that the runtime loads in place of the one it has in the main runtime DLL (coreclr.dll). This DLL needs to reside in the same directory as *coreclr.dll*
```

Comment:
```suggestion
- Specifies the name of a standalone GC DLL that the runtime loads in place of the one it has in the main runtime DLL (coreclr.dll). This DLL needs to reside in the same directory as *coreclr.dll*.
```

Reply:
Maybe you were trying to be polite, but I would not have minded if you had updated the PR directly in GH. (assuming you know you can do it, even on forks)

Reply:
Yes, I wanted to be polite but would have eventually committed it if necessary. Thanks!